### PR TITLE
Temporal new Quickstart.

### PR DESCRIPTION
**Pull request by pkudikyala**
**Ticket:** https://new-relic.atlassian.net/browse/NR-229976
**Description:** Created Temporal new quickstart.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: alert-policies/temporal/ActivityExecutionLatency.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Activity Excution Latency
```
Comment (mdumpati) [inline alert-policies/temporal/ActivityExecutionLatency.yml:2]:
```suggestion
name: Activity Execution Latency
```

File: alert-policies/temporal/ActivityEexcutionFail.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Name of the alert
+name: Activity Execution Fails
```
Comment (mdumpati) [inline alert-policies/temporal/ActivityEexcutionFail.yml:2]: @pkudikyala In filename did spelling mistake can you update it?

**Review by mdumpati — COMMENTED** [review]
@pkudikyala Check my comments

**Review by mdumpati — DISMISSED** [review]
LGTM

File: alert-policies/temporal/ActivityExecutionFail.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Name of the alert
+name: Activity Execution Fails
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal activity execution fails 1 for 5 minutes.
```
Comment (seemantk) [inline alert-policies/temporal/ActivityExecutionFail.yml:6]:
```suggestion
  This alert is triggered if the Temporal activity execution fails once within a 5 minutes window.
```

File: alert-policies/temporal/ActivityScheduleStartLatency.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Activity Schedule to Start Latency
+
+# Description and details
+description: |+
+  This alert is triggered if the Temporal activity schedule to start latency exceeds 5 seconds for 5 minutes.
```
Comment (seemantk) [inline alert-policies/temporal/ActivityScheduleStartLatency.yml:6]: Unsure how to understand this. Can you please clarify this?

Comment (pkudikyala) [reply alert-policies/temporal/ActivityScheduleStartLatency.yml:6]: Hi @seemantk,
The Temporal Schedule-To-Start Timeout represents the maximum allowable duration between scheduling an activity task (placing it in a task queue) and a worker starting (picking it up from the task queue). Essentially, it serves as a restriction on the length of time an activity task can remain in the queue.

File: alert-policies/temporal/ActivityExecutionFail.yml (hunk 3 above)
Comment (pkudikyala) [reply alert-policies/temporal/ActivityExecutionFail.yml]: Hi @seemantk ,
I made the review comment.

File: alert-policies/temporal/ActivityEexcutionFail.yml (hunk 2 above)
Comment (pkudikyala) [reply alert-policies/temporal/ActivityEexcutionFail.yml:2]: Resolved review comments

**Review by mdumpati — DISMISSED** [review]
LGTM

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM!

**Pull request merged**